pom.xml

```diff
@@ -22,6 +22,8 @@
     <groupId>io.confluent</groupId>
     <artifactId>docker-utils</artifactId>
     <packaging>jar</packaging>
+    <name>docker-utils</name>
+    <version>5.1.0-SNAPSHOT</version>
```

Comment:
Why aren't we inheriting from parent here?

Reply:
What you mean by having it inherited from parent?
Do you mean you want me to pass ${project.version}?

Reply:
@aayars I see that common/pom has the following property available:
https://github.com/confluentinc/common/blob/05138536407fa6b497e48fd6d56b48a58fa20488/pom.xml#L51

I can use confluent.version instead, which will come from parent.